packages/landing-site/src/components/resources-info-banner.tsx

```diff
@@ -3,18 +3,18 @@ import { sprinkles } from "@framework/system/src/sprinkles/sprinkles.css"
 
 export default function ResourcesInfoBanner() {
 	const cardResources = [
-		{
-			title: "React Resources",
-			href: "https://react.framework.dev/",
-			icon: "/icon-react.svg",
-			backgroundColor: "#00BCDA",
-		},
 		{
 			title: "Angular Resources",
 			href: "https://angular.framework.dev/",
 			icon: "/icon-angular.svg",
 			backgroundColor: "#C3002F",
 		},
+		{
+			title: "React Resources",
+			href: "https://react.framework.dev/",
+			icon: "/icon-react.svg",
+			backgroundColor: "#00BCDA",
+		},
```

Comment:
Noticed this exact block is used in 2 places. Can we make this a global export for both sections to avoid duplication and to make it easier to add in the future?